src/server/repository/public/user/getMany.ts

```diff
@@ -2,9 +2,10 @@ import { Objects } from '@utils/objects'
 
 import { CountryIso } from '@meta/area'
 import { Assessment, Cycle } from '@meta/assessment'
-import { CollaboratorProps, RoleName, User } from '@meta/user'
+import { RoleName, User } from '@meta/user'
 
 import { BaseProtocol, DB } from '@server/db'
+import { UserRoleAdapter } from '@server/repository/adapter'
 
 import { fields } from './fields'
```

Comment:
check if fields must be updated

Reply:
I meant user role props, if permissions are included and props are excluded  in select

Reply:
excluded from query

`jsonb_agg(to_jsonb(ur.* ) - 'props')`